content/blog/2021/01/2021-01-12-new-year-report.adoc

```diff
@@ -0,0 +1,248 @@
+---
+layout: post
+title: "2020 A Year Like No Other"
+tags:
+- community
+- new-year-blogpost
+- governance
+authors:
+- markyjackson-taulia
+- oleg_nenashev
```

Comment:
Anyone else? CC @MarkEWaite

Reply:
That list seems reasonable to me.  The others were reviewers more than authors.